PlayerCollection.cs

```diff
@@ -106,7 +121,9 @@ public class PlayerDisplay
 
         public string CharacterName => _player.CharacterName;
         public bool IsFormerSquadMember => _detailsButton.Parent?.Equals(_formerPlayerPanel) ?? false;
+        public bool HasChangedCharacters => _player.HasChangedCharacters;
         public bool IsSelf => _player.IsSelf;
+        public string AccountName => _player.AccountName;
```

Comment:
Remark: I think, if we continue to expose every `Player` field here, it may be better to directly expose the `Player` object in readonly acces!